lib/HTFeed/Stage/ImageRemediate.pm

```diff
@@ -371,6 +374,13 @@ sub _remediate_tiff {
 	$self->{newFields}
     );
 
+    my $end_time   = $self->{job_metrics}->time;
+    my $delta_time = $end_time - $start_time;
+    $self->{job_metrics}->add("ingest_imageremediate_seconds", $delta_time);
+    $self->{job_metrics}->inc("ingest_imageremediate_images");
+    $self->{job_metrics}->inc("ingest_imageremediate_bytes_read", $infile_size);
```

Comment:
I think you want `add` rather than `inc` here - `inc` always increments by 1.

Reply:
Yep. Good catch. All other hits when grepping for `'inc\(.+s'` are for items and images, as they should.